src/bar/BarChartWithErrors.ts

```diff
@@ -58,6 +62,32 @@ export default class BarChartWithErrors extends mixins(Bar) {
                         if (tooltipItem.yLabel) {
                             label += formatCallback(tooltipItem.yLabel);
                         }
+
+                        interface ErrorBars {
+                            [xLabel: string]: {
+                                minus: number
+                                plus: number
+                            }
+                        }
+
+                        interface ChartDataSetsWithErrors extends ChartDataSets {
+                            errorBars?: ErrorBars
+                        }
+
+                        let minus = null
+                        let plus = null
+
+                        if (this.showErrors && tooltipItem.xLabel && typeof tooltipItem.datasetIndex !== "undefined" && data.datasets && data.datasets[tooltipItem.datasetIndex]) {
+                            const errorBars = (data.datasets[tooltipItem.datasetIndex] as ChartDataSetsWithErrors).errorBars
+                            if (errorBars && errorBars[tooltipItem.xLabel]) {
+                                minus = errorBars[tooltipItem.xLabel].minus
+                                plus = errorBars[tooltipItem.xLabel].plus
+                            }
+                        }
+                        if (minus && plus) {
```

Comment:
Might want to be more explicit than a falsy check here as presumably 0 would be a valid minus value?